### PR TITLE
added link to the Whitepaper on iota.org

### PR DESCRIPTION
**Pull request by Phyloiota**

### CODE REVIEW

File: documentation/docs/guide/core_concepts/iscp-architecture.md [hunk 1]
```diff
@@ -5,18 +5,18 @@ keywords:
...
-# ISCP Architecture
+# IOTA Smart Contracts Architecture
 
-With ISCP, anyone can start their own chain and define the validators.
+With IOTA Smart Contracts, anyone can start their own chain and define the validators.
 
 Each chain has its own state where a state update (going from one block to the next) is hashed and published to the Tangle, which moves the state anchor on Layer 1.
 
-The multi-chain nature of ISCP makes it a more complex implementation of smart contracts, over say Ethereum, as illustrated here:
+The multi-chain nature of IOTA Smart COntracts makes it a more complex implementation of smart contracts, over say Ethereum, as illustrated here:
```
Comment (grueegg) [inline documentation/docs/guide/core_concepts/iscp-architecture.md]: Typo in Contracts --> O should be lower case

**Review by grueegg — CHANGES_REQUESTED** [review]

**Comment by charlesthompson3** [comment]
Refers to points 3 and 7 in https://github.com/iotaledger/docs.iota.org/issues/58

**Review by Phyloiota — COMMENTED** [review]
corrected the typo, thanks

**Review by fijter — APPROVED** [review]
Thanks! :)

**Pull request merged**